resources/assets/js/components/Application/Skills/Skills.tsx

```diff
@@ -0,0 +1,623 @@
+              }}
+            />
+          </p>
+          <p data-c-margin="bottom(.5)">
+            <FormattedMessage
+              id="application.skills.instructionListStart"
+              defaultMessage="Try answering one or two of the following questions:"
+              description="Paragraph before the instruction list on the Skills step."
+            />
+          </p>
+          <ul data-c-margin="bottom(1)">
+            <FormattedMessage
+              id="application.skills.instructionList"
+              defaultMessage="<bullet>What did you accomplish, create, or deliver using this skill?</bullet><bullet>What tasks or activities did you do that relate to this skill?</bullet><bullet>Were there any special techniques or approaches that you used?</bullet><bullet>How much responsibility did you have in this role?</bullet>"
+              description="List of potential justification helpers on the Skills step."
+              values={{
+                bullet: (chunks) => <li>{chunks}</li>,
+              }}
+            />
+          </ul>
+          <p>
+            <FormattedMessage
+              id="application.skills.instructionListEnd"
+              defaultMessage="If a skill is only loosely connected to an experience, consider removing it. This can help the manager focus on your best examples."
+              description="Paragraph after the instruction list on the Skills step."
+            />
+          </p>
+          <div className="skills-list">
+            {criteria.map((criterion) => {
+              const skill = getSkillOfCriteria(criterion, skills);
+              if (skill === null) {
+                return null;
+              }
+              const skillName = localizeFieldNonNull(locale, skill, "name");
+              const skillDescription = localizeFieldNonNull(
+                locale,
+                skill,
+                "description",
+              );
+              const skillHtmlId = slugify(skillName);
+
+              return (
+                <>
+                  <h3
+                    className="application-skill-title"
+                    data-c-heading="h3"
+                    data-c-padding="top(3) bottom(1)"
+                    data-c-margin="bottom(1)"
+                    id={skillHtmlId}
+                  >
+                    <button
+                      data-c-font-size="h3"
+                      data-c-dialog-id={modalId}
+                      type="button"
+                      onClick={(e): void => {
+                        setModalHeading(skillName);
+                        setModalBody(skillDescription);
+                        setVisible(true);
+                      }}
+                    >
+                      {skillName}
+                    </button>
+                    <br />
+                    <a
+                      data-c-font-size="normal"
+                      data-c-font-weight="bold"
+                      href="/faq#levels"
```

Comment:
Wasn't sure about routing for other pages outside of the client side routes, so I have this for now

Reply:
I'd add an applicant equivalent to `managerFaq` in `helpers/routes.ts`